dolphinscheduler-graphql/mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
There is no point to add maven wrapper again, in this module, it's already added in the root project